README.md

```diff
@@ -56,3 +56,5 @@ Some icons made by [Freepik](https://www.freepik.com) from [www.flaticon.com](ht
   [3]: https://github.com/ktemkin/qemu/blob/with_tcti/tcg/aarch64-tcti/README.md
   [4]: https://github.com/ish-app/ish
   [5]: https://github.com/holzschu/a-shell
+  
+[ZIP Foundation](https://github.com/weichsel/ZIPFoundation) is released under the MIT License.
```

Comment:
Can you move this above the links and below `Some icons...`? The way markdown is usually formatted is that you put references at the bottom of the document (unless they're inline).